test/mpi/test_mpi_cuda.c

```diff
@@ -474,7 +474,11 @@ static int test_alloc_prim_send_user(const test_params_t *params)
     alloc_mem(params, &alloc_mem_send, &alloc_mem_recv);
 
     CUDA_CHECK(cuCtxPopCurrent(&primary_ctx));
+#if CUDA_VERSION >= 13000
+    CUDA_CHECK(cuCtxCreate(&user_ctx, NULL, 0, params->cu_dev));
+#else
     CUDA_CHECK(cuCtxCreate(&user_ctx, 0, params->cu_dev));
+#endif
```

Comment:
I added this function in https://github.com/openucx/ucx/pull/10670.
I think `cuCtxCreate_v4` is the latest version, and was added in 12.5.
```C
static CUresult cuCtxCreate_versioned(CUcontext *ctx, CUdevice cu_dev)
{
#if CUDA_VERSION >= 12050
    CUctxCreateParams ctx_create_params = {};
    return cuCtxCreate_v4(ctx, &ctx_create_params, 0, cu_dev);
#else
    return cuCtxCreate(ctx, 0, cu_dev);
#endif
}
```
https://github.com/openucx/ucx/pull/10670 hasn't been reviewed yet.